keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/DoubleVertex.java

```diff
@@ -58,6 +59,10 @@ public DoubleVertex(long[] initialShape) {
         super(initialShape);
     }
 
+    public DoubleVertex print() {
```

Comment:
We should also have a `print(PrintStream)` method here

Reply:
This has the same functionality as the print on the `Vertex` class so I think you don't need to override. See my comment on `Vertex.java` for something that should work.